frame/support/src/storage/bounded_vec.rs

```diff
@@ -35,14 +35,14 @@ use crate::{
 /// It has implementations for efficient append and length decoding, as with a normal `Vec<_>`, once
 /// put into storage as a raw value, map or double-map.
 ///
-/// As the name suggests, the length of the queue is always bounded. All internal operations ensure
-/// this bound is respected.
+/// The length of the vec is not strictly bounded. Decoding a vec with more element that the bound
+/// is accepted, and some method allow to bypass the restriction with warnings.
```

Comment:
Why not bound the decode?

Reply:
I would also prefer so, but it seems people want to use it as a lightly bounded vec https://github.com/paritytech/substrate/issues/8839